server.js

```diff
@@ -100,11 +100,11 @@ app.use('/api', function (req, res) {
     const service = opbeansServiceUrls[Math.floor(Math.random() * opbeansServiceUrls.length)]
     opts.hostname = service.hostname
     opts.port = service.port
-    opts.path = req.path
+    opts.path = req.originalUrl
   } else {
     opts.hostname = conf.server.hostname
     opts.port = conf.server.port2
-    opts.path = req.path.replace(/^\/api/, '')
+    opts.path = req.url // req.url will just contain the sub-path without the /api
```

Comment:
The routes in `api.js` would also need to be mounted at `/api` for this to work. https://github.com/elastic/opbeans-node/blob/master/api.js#L59

Reply:
No if the real url us `/api/foo`, the `req.url` will just contain `/foo` because Express overwrites it with the sub-app url when you use `app.use('/api', ...)`

Reply:
Ah, sorry, was misunderstanding that this would be the full-url here and the API server would receive requests with the `/api` prefix, which it won't serve.